soundbay/utils/metadata_processing.py

```diff
@@ -0,0 +1,186 @@
+from pathlib import Path
+import pandas as pd
+import numpy as np
+import re
+import soundfile as sf
+from typing import List
+
+
+# TODO add tests to the utils in this file
+
+# <<<<<<< feature/EDA_script
+def raven_to_df_annotations(annotations_path: str,
+                            recording_path: str,
+                            positive_tag_names: list = ['w', 'sc']):
+    """
+    Takes annotation files (selection table) created in Raven and turns it to a compatible annotations csv.
+    """
+    # create dataframe
+    annotations = Path(annotations_path)
+    recording = Path(recording_path)
+    # ignore irrelevent files
+    filelist = list(annotations.glob('*selections.txt'))
+    metadata = []
+    for file in filelist:
+        dfTemp = pd.read_csv(file, sep="\t")
+        dfTemp['filename'] = re.search(
+            "\.Table.1.selections", file.as_posix()).group()
+        dfTemp['FirstCallTime'] = np.amin(dfTemp['Begin Time (s)'])
+        dfTemp['LastCallTime'] = np.amax(dfTemp['End Time (s)'])
+        metadata.append(dfTemp)
+
+    metadata = pd.concat(metadata)
+    metadata.rename(columns={'Begin Time (s)': 'begin_time', 'End Time (s)': 'end_time'}, inplace=True)
+    print('Number of Labels:', metadata.shape[0])
+    for tag in positive_tag_names:
```

Comment:
what does this loop do?

Reply:
no idea, `raven_to_df_annotations` is an old leftover from utils. 
Perhaps we should delete it and create the moz dataset using the smaller new util functions.